src/index.html

```diff
@@ -2,11 +2,383 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta 
+      name="viewport" 
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,900&display=swap" rel="stylesheet"
+    >
+    
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="moyo-logo">
+      </a> 
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+  
+          <li class="nav__item">
+            <a class="nav__link" href="#Samsung">Samsung</a>
+          </li>
+  
+          <li class="nav__item">
+            <a class="nav__link" href="#Smartphones">Smartphones</a>
+          </li>
+  
+          <li class="nav__item nav__item--blue">
+            <a 
+              data-qa="nav-hover"
+              class="nav__link" 
+            href="#Laptops & Computers" 
+            >
```

Comment:
please make sure all attributes have similar distance
```suggestion
            <a 
              data-qa="nav-hover"
              class="nav__link" 
              href="#Laptops & Computers" 
            >
```